main.tf

```diff
@@ -1,15 +1,16 @@
 module "aws_deploy-dev1-eu-west-2" {
-  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=v1.0.0"
+  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=v2.1.0"
   env               = "dev1"
   bootstrap_version = "${var.bootstrap_version}"
   vault_role        = "ae-node"
   vault_addr        = "${var.vault_addr}"
 
-  spot_nodes       = 3
-  spot_price       = "0.125"
-  instance_type    = "m4.large"
-  ami_name         = "aeternity-ubuntu-16.04-v1549009934"
-  root_volume_size = 20
+  spot_nodes_min = 3
+  spot_nodes_max = 3
+
+  spot_price    = "0.125"
```

Comment:
i guess this could be lowered as it is ~0.03-0.04

Reply:
https://github.com/aeternity/terraform-aws-devnet/pull/2/commits/5c1ac5e7860ea34acc76e8018fc9e2174b8a557e